### ISSUE TITLE
Spanish Translation

### ISSUE THREAD

**Issue by CodeKommissar** [getify/Functional-Light-JS#115]
Hi Kyle, the other day I saw on the @FLJS twitter account that the book was almost ready, I was wondering if I was OK for me to start translating the book to spanish so that more people could use it. I saw the [Chinese translation complete issue](https://github.com/getify/Functional-Light-JS/issues/106) and I'd like to do something similar (mirroring the repo with a fork and start translating from there), tell me what you think

**Comment by getify** [comment]
Yep go for it! Make sure when you are done you come back here so we can merge the translation back into the main repo. :)

**Comment by CodeKommissar** [comment]
Got it, thanks! Yeah, I will do.
